### ISSUE TITLE
pre-index and access-config not loading properly after some time

### ISSUE THREAD

**Issue by Matzexxxxx** [whitewillem/PMSF#135]
**Issue:**
When using an access-config and discord-login, the permissions to the logged in discord-user isnt working properly after some time ( maybe 5-15 minutes after closing and reopening the map)
For my case, the frontend is only showing the default config enabled options, but not the ones defined in the access-config (for the pokemon-menuoption for example).
Something is messing up with the session i guess, since the logged in user is still displayed in the navbar, but everything from the access-config isnt applied.
Also, the modt defined in the access-config is displayed when that error occurs. even if the modt is set to false in the access-config, but true in the normal configs.

**Reproduce that error:**
Setup a PMSF instance with a proper discord-login and an access-config.
disable modt in the access-config and enable it in the normal config.php
enable the pokemon menu (noPokemon = false) in the access-config and disable the pokemon menu by default config.php (noPokemon = true)
Log in on the map, close the browser - wait 10-15 minutes and reopen the map again. 

**PS:**
On that status, reloading the page again is resolving that problem. After the reload, everything is fine.
Did anyone else experienced this? its only with access-config and discord-login. Without the login-system there is no such an error

**Comment by Soda-City-PoGo** [comment]
I experience the same issue, yes refreshing works to fix it. I thought it always had something to do with the anti-scraping stuff but i don't know.

**Comment by xxleevo** [comment]
This also seems to happen after you reopen your browser. Time doesnt matter
If you only close the tab and open the map in a new one, its like a reload and fixing it. If you close the browser and reopen up the browser, the error is there again.

**Comment by gtkingz** [comment]
I can confirm I have the same problem.

**Comment by ColdShadow80** [comment]
Problem is not exclusive to Discord login. Have same problem in legacy (email) login.
Basicaly, after loggin in you need to refresh to get all menu data.

Example, if no login users have no acesso to "Active quests" menu and it is disable by default, the menu is hidden. After login in a profile that has acess, the menu still has the option hidden.

Only after a manual refresh are the setinngs fully aplied and the side menu refreshed to the proper acess-config settings.

**Comment by plinytheelder** [comment]
This has been broken since the login system was merged into PMSF a year ago. It is currently the biggest bug plaguing our users and our inboxes from users who complain or don't understand why their access isn't working correctly.

**Comment by t00fan** [comment]
This is a pretty big pain point for the users.  Creates lots of inquiry for access.  Would really appreciate a fix! Thank you.

**Comment by Gladiator10864** [comment]
Please fix. I'll bake you cookies and ship if you do!!! This is the number one complaint I get from my users on the map as well.

**Comment by ColdShadow80** [comment]
Problem seems solved with latest dev commits. Tested using MAD branch and RM database

**Comment by whitewillem** [comment]
Will close this for now as its 'fixed' in develop branch.

**Comment by DrKendall** [comment]
Similar issue has arose after closing the browser. 
When the browser is closed after login, you're logged out when you revisit. Even if I revisit within 10s of closing.
